### ISSUE TITLE
[BUG] — The last video of a playlist is missing

### ISSUE THREAD

**Issue by Asgarrrr** [SuspiciousLookingOwl/youtubei#27]
**Describe the bug**
The last video of all the playlist I tested is missing

**My code**
```js

// —— I always tested on playlists with less than 30 items, and using ".next()" only returns an empty array
const test = await youtube.getPlaylist( query )

console.log( test.videoCount );
console.log( test.videos );

```

**Screenshots**

<img width="1143" alt="Capture d’écran 2021-09-13 à 05 50 42" src="https://user-images.githubusercontent.com/36574651/133021326-99386917-f444-4c56-aa9a-e82c6b133425.png">

**Comment by SuspiciousLookingOwl** [comment]
Thank you for reporting this, should be fixed on `0.0.1-rc.23`

**Comment by Asgarrrr** [comment]
All right! Thanks for your work ( and good luck )
